src/envattrs/__init__.py

```diff
@@ -1,17 +1,35 @@
 import os
 from typing import TypeVar, Type
 
-import attr
+try:
+    import attr
+except ImportError:
+    try:
+        import dataclasses
+    except ImportError:
+        print("Neither dataclasses not attr libraries available")
+        print("Make sure you are using either one and install that module")
 
-T = TypeVar('T')
+
+T = TypeVar("T")
 
 SubAttrs = object()
 
 
 def _iter(cls, prefix, source):
-    for field in attr.fields(cls):
+    # Check if it is an attr class or dataclass
+    # and use the corresponding fields function
+    if attr.has(cls):
+        fields = attr.fields
+    elif dataclasses.is_dataclass(cls):
+        fields = dataclasses.fields
+    else:
+        print("Object is neither dataclass nor attr class")
+        return
```

Comment:
This should raise an exception (probably `TypeError` ?)

Reply:
I think it should be fixed now.